net.mkiol.Jupii.yaml

```diff
@@ -0,0 +1,321 @@
+app-id: net.mkiol.Jupii
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+rename-icon: jupii
+command: jupii
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all # v4l2 webcams
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-videos
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.Tracker1.Steroids # media files index
+  - --talk-name=org.freedesktop.Tracker3.Endpoint # media files index
+  - --talk-name=org.kde.StatusNotifierWatcher # system tray icon
+  - --env=QT_QUICK_CONTROLS_STYLE=org.kde.desktop
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - "*/cmake/*"
+  - "*.a"
+  - "*.la"
+modules:
+  - name: taglib
```

Comment:
https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/qt5.15lts-23.08/org.kde.Sdk.json.in?ref_type=heads#L1254 it's in runtime

Reply:
Indeed! Fixed in 10300505e68e10ce669075930fa3ffea984acb50